libraries/cloudharness-common/cloudharness/auth/keycloak.py

```diff
@@ -250,9 +250,14 @@ def create_client_role(self, client_id, role):
         return True
 
     @with_refreshtoken
-    def get_group(self, group_id, with_members=False) -> UserGroup:
+    def get_group(self, group_id, with_members=False, brief_representation=True) -> UserGroup:
```

Comment:
The api looks a little bit inconsistent as there is already the `with_members` attribute. It would be more natural a parameter like `with_attributes` with default `false`

Reply:
it's not only attributes :-), like realroles and client roles.... we could do `with_details` what do you think?

Reply:
looks good

Reply:
done